warehouse/query-core/src/main/java/datawave/next/async/ContextualUncaughtExceptionHandler.java

```diff
@@ -0,0 +1,16 @@
+package datawave.next.async;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ContextualUncaughtExceptionHandler implements Thread.UncaughtExceptionHandler {
```

Comment:
not sure this is necessary if linked to the existing uncaughtExceptionHandler associated with the query

Reply:
Created https://github.com/NationalSecurityAgency/datawave/issues/2910